catalog/transformer/catalog-transformer-xml/pom.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- 
+<!--
```

Comment:
It's probably best to undo the changes to this file since it's not related to the HTML transformer.